src/model/request/BurnEthErc20.ts

```diff
@@ -0,0 +1,14 @@
+import {
+    IsIn,
+    IsInt,
+    IsNotEmpty,
+    IsNumberString,
+    IsOptional,
+    Length,
+    Min,
+} from 'class-validator';
+import { Currency } from './Currency';
+import { BurnErc20 } from './BurnErc20'
+
+export class BurnEthErc20 extends BurnErc20 {
```

Comment:
this class is not needed. we can use Base class for ETH

Reply:
ok